content/docs/04.5-devx/05-app-profile/08-services/50-service-listings/2-postgresql-db.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Postgres"
+metaTitle: "Palette Dev Engine Postgres Service"
+metaDescription: "Palette Dev Engine Postgres Service"
+hideToC: false
+type: "appTier"
+category: ['databases']
+fullWidth: false
+logoUrl: 'https://upload.wikimedia.org/wikipedia/commons/2/29/Postgresql_elephant.svg'
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+
+# Postgres
+
+Palette supports [Postgres](https://www.postgresql.org/) as a database service. Postgres is a powerful open-source object-relational database system with over 35 years of active deployment with a strong reputation for reliability, feature robustness, and performance. Postgres uses and extends the SQL language combined with many features that safely store and scale the most complicated data workloads.
+
+<br />
+
+## Palette Postgres Requirement
+
+The following are requirements for using Palette Postgres:
+
+* Do not use the Postgres user names, `postgres` and `admin`. These user names are reserved for internal system operations and will cause internal conflicts.
+
+
+* The user name format does not support the special character hyphen(-). Example: `name-1` is not supported. 
+
+
+* Clients must set `sslMode=require` or greater as the server instance requires encryption for all connections. Review the [Postgres SSL documentation](https://www.postgresql.org/docs/current/libpq-ssl.html) to learn more about the SSL modes.
```

Comment:
```suggestion
* Clients must set `sslMode=require` or greater, as the server instance requires encryption for all connections. Review the [Postgres SSL documentation](https://www.postgresql.org/docs/current/libpq-ssl.html) to learn more about the SSL modes.
```

Reply:
'or greater' in the first sentence is confusing to me - or greater than 'require'?